contracts/contracts/sbtc-registry.clar

```diff
@@ -44,10 +48,16 @@
   }
 )
 
-;; Read-only functions
+;; Data structure to store aggregate pubkey,
+;; stored to avoid replay
+(define-map aggregate-pubkeys (buff 32) bool)
 
+;; Data structure to store the current signer set,
+;; stored to avoid replay
+(define-map multi-sig-address principal bool)
+
+;; Read-only functions
```

Comment:
Same thing here with doc comments. 

I know we don't have an internal process for generating contract documentation yet, so I'm fine with punting this and I can open a ticket to clean up doc comments and also setup a process for auto-generated documentation.

Reply:
Copy let's punt on documentation formatting.